### PR TITLE
FIX: topomap plot from certain layouts

### PR DESCRIPTION
**Pull request by dengemann**
topomap currently broken with e.g. magnesWH3600 layout. this is the fix.

### CODE REVIEW

**Comment by dengemann** [comment]
cc @agramfort @wmvanvliet

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def _prepare_topo_plot(inst, ch_type, layout):
         # change names so that vectorview combined grads appear as MEG014x
         # instead of MEG0142 or MEG0143 which are the 2 planar grads.
         ch_names = [ch_names[k][:-1] + 'x' for k in range(0, len(ch_names), 2)]
-
+    pos = np.array(pos)[:, :2]  # 2D plot, otherwise interpolation bugs
```
Comment (agramfort) [inline mne/viz/topomap.py]: pos size is not consistent across data types?

Comment (dengemann) [inline mne/viz/topomap.py]: it apparently has 4 columns. Whaterv is going on, here only 2 of them should be used.

Comment (dengemann) [inline mne/viz/topomap.py]: I think it used to be different before, but too lazy to go through all git blame. Probably it was @wmvanvliet ;-)

**Comment by agramfort** [comment]
alright

**Pull request merged**